add telemetry plugin (#608)
cap-js/docs: plugins/index.md

Click on the icon to get detailed instructions. {.learn-more}

## Telemetry (Beta)

[@cap-js/telemetry](https://npmjs.com/package/@cap-js/telemetry) {.subtitle}

The Telemetry plugin provides observability features such as tracing and metrics, including [automatic OpenTelemetry instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/automatic).

> Simply add the plugin to your project and you will find telemetry output written to the console like so:

Reviewer: ```suggestion
Simply add the plugin to your project and you will find telemetry output written to the console as follows:
```

```txt
[odata] - GET /odata/v4/processor/Incidents
[telemetry] - elapsed times:
  0.00 → 2.85 = 2.85 ms  GET /odata/v4/processor/Incidents
  0.47 → 1.24 = 0.76 ms    ProcessorService - READ ProcessorService.Incidents
  0.78 → 1.17 = 0.38 ms      db - READ ProcessorService.Incidents
  0.97 → 1.06 = 0.09 ms        @cap-js/sqlite - prepare SELECT json_object('ID',ID,'createdAt',createdAt,'creat…
  1.10 → 1.13 = 0.03 ms        @cap-js/sqlite - stmt.all SELECT json_object('ID',ID,'createdAt',createdAt,'crea…
  1.27 → 1.88 = 0.61 ms    ProcessorService - READ ProcessorService.Incidents.drafts
  1.54 → 1.86 = 0.32 ms      db - READ ProcessorService.Incidents.drafts
  1.74 → 1.78 = 0.04 ms        @cap-js/sqlite - prepare SELECT json_object('ID',ID,'DraftAdministrativeData_Dra…
  1.81 → 1.85 = 0.04 ms        @cap-js/sqlite - stmt.all SELECT json_object('ID',ID,'DraftAdministrativeData_Dr…
```

> Additional to the default console output, there are predefined kinds for exporting telemetry data to Dynatrace and Jaeger, with [SAP Cloud Logging](https://help.sap.com/docs/cloud-logging) to follow shortly.

Reviewer: ```suggestion
In addition to the default console output, there are predefined kinds for exporting telemetry data to Dynatrace and Jaeger.  The [SAP Cloud Logging](https://help.sap.com/docs/cloud-logging) service is coming soon.
```
